src/datastore/postgres-store.ts

```diff
@@ -3011,6 +3011,57 @@ export class PgDataStore
     return tx;
   }
 
+  async getMempoolTxs(args: {
+    txIds: string[];
+    includeUnanchored: boolean;
+    includePruned?: boolean;
+  }) {
+    return this.queryTx(async client => {
+      const conditionParam = this.generateParameterizedWhereOrClause('tx_id', args.txIds.length);
```

Comment:
Looks like usages of `generateParameterizedWhereOrClause` can be replaced using `WHERE tx_id = ANY($1)` syntax, which would result in more readable and performant code

Reply:
Nice catch. Changed.